source/extensions/common/wasm/wasm.cc

```diff
@@ -497,6 +497,10 @@ Wasm* PluginConfig::maybeReloadHandleIfNeeded(SinglePluginHandle& handle_wrapper
     return wasm;
   }
 
+  // Publish RuntimeError event when detected. On failure, only the state is updated by sdk, no
```

Comment:
Two questions:
1) Are you certain no events can happen besides the runtime one here? What other errors are possible here?

2) I think this function is called repeatedly every time a wasm context needs to be associated by a filter? That would make the stat increment even though it only crashed once?